packages/backend/src/payment-method/handler/service.ts

```diff
@@ -20,8 +21,16 @@ export interface PaymentQuote {
   additionalFields: Record<string, unknown>
 }
 
+export interface PayOptions {
+  receiver: Receiver
+  outgoingPayment: OutgoingPayment
+  finalDebitAmount: bigint
+  finalReceiveAmount: bigint
```

Comment:
these are called 'finalX' because the amount is adjusted in the outgoing payment service before calling ILP pay (due to retries, for example, where in the case of ILP pay, we've already sent some amount of currency)